src/slangpy_ext/utils/slangpytensor.cpp

```
@@ -689,6 +1102,136 @@ nb::object NativeTensorMarshall::create_dispatchdata(nb::object data) const
+    nb::object data,
+    nb::object result
+) const
+{
+    SGL_UNUSED(context);
+    SGL_UNUSED(binding);
+
+    auto ndarray = nb::cast<nb::ndarray<nb::numpy>>(data);
+    auto tensor = nb::cast<NativeTensor*>(result);
+
+    size_t numpy_data_size = ndarray.nbytes();
+    size_t buffer_data_size = tensor->storage()->size();
```

Cppcheck (2.20.0)

[style] The function 'get_hashed_string' is never used.

(unusedFunction)